### PR TITLE
UI/UX Design category - Github Repositories 

### LINKED ISSUE #316: 💡 [FEATURE] - Usefull Github repos for UI/IX design
### Idea Contribution

- [X] I have read all the feature request issues.
- [X] I'm interested in working on this issue
- [X] I'm part of GSSOC organization

### Explain feature request

Github repos can be a valuable resources for learning UI/UX but there is no github repos table in UI/UX design category.

### Explain your solution

I would like to add few helpfull github repos under UI/UX category.

### Any alternative approaches/features

_No response_

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by chaanakyaaM**
**IMPORTANT:** When creating your Pull Request, please use the following format for the title: `Category - Resource Name & Section`  

PR Title samples:

- `Web development - CSS Portal & Tools`
- `Web development - Leetcode & Challenges`

Closes #316 

Five useful github repositories added to UI/UX Design category.

```
### GitHub Repositories

>These GitHub repositories help you Learn UI/UX Designs efficiently.

<table width="100%">
    <tr>
      <th>Resource Name</th>
      <th>Description</th>
    </tr>
    <tr>
      <td width="150px"><a href="https://github.com/Pediomo/UI-UX-resources">UI-UX Resources</a></td>
      <td>The GitHub repository UI-UX-resources Created and managed by Pediomo offers a comprehensive list of resources related to UI/UX design. It includes articles, books, videos, tools, podcasts, courses, and more.</td>
    </tr>
    <tr>
      <td width="150px"><a href="https://github.com/hendurhance/ui-ux">UI-UX</a></td>
      <td>The GitHub repository UI-UX is created and managed by  hendurhance is a collection of resources which includes various tools, frameworks, and guidelines to enhance the design process and create better user experiences.</td>
    </tr>
    <tr>
      <td width="150px"><a href="https://github.com/togiberlin/ui-ux-designer-roadmap?tab=readme-ov-file">UI-UX Designer roadmap</a></td>
      <td>The "UI/UX Designer Roadmap" GitHub repository by togiberlin provides a solid understanding of UI-UX designer's roadmap and it also includes clear learning path, detailing essential skills and tools necessary for becoming proficient in UI/UX design.</td>
      </tr>
    <tr>
      <td width="150px"><a href="https://github.com/gregjw/ui-ux">UI-UX</a></td>
      <td>The GitHub repository "UI-UX" by gregjw is a curated list of resources for UI/UX designers. It includes categories such as user research tools, standards and guidelines, prototyping tools, browser extensions, design tools,various frameworks and more.</td>
      </tr>
      <tr>
      <td width="150px"><a href="https://github.com/greyhatguy007/meta-front-end-developer-professional-certificate">Meta frontend developer professional course</a></td>
      <td>The GitHub repository "Meta Front-End Developer Professional Certificate" by greyhatguy007, this repository is organized into various courses, covering topics such as HTML, CSS, JavaScript, React, version control, and advanced React concepts. </td>
      </tr>
</table>   
```

### Checklist:
Before submitting your pull request, ensure that you have completed the following tasks: 

- [x] I have carefully reviewed and adhered to the [contributing guidelines](https://github.com/jfmartinz/ResourceHub/blob/main/CONTRIBUTING.md) before creating this pull request.
- [x] I followed the prescribed PR title template. _(Check this if you're adding a resource)_

> fill the bracket with x. Something like this [x]

### CODE REVIEW

File: UI-UX Design/readme.md [hunk 1]
```diff
@@ -614,5 +615,35 @@ exercises.</td>
...
+      <td>The "UI/UX Designer Roadmap" GitHub repository by togiberlin provides a solid understanding of UI-UX designer's roadmap and it also includes clear learning path, detailing essential skills and tools necessary for becoming proficient in UI/UX design.</td>
+      </tr>
+    <tr>
+      <td width="150px"><a href="https://github.com/gregjw/ui-ux">UI-UX</a></td>
+      <td>The GitHub repository "UI-UX" by gregjw is a curated list of resources for UI/UX designers. It includes categories such as user research tools, standards and guidelines, prototyping tools, browser extensions, design tools,various frameworks and more.</td>
+      </tr>
+      <tr>
+      <td width="150px"><a href="https://github.com/greyhatguy007/meta-front-end-developer-professional-certificate">Meta frontend developer professional course</a></td>
+      <td>The GitHub repository "Meta Front-End Developer Professional Certificate" by greyhatguy007, this repository is organized into various courses, covering topics such as HTML, CSS, JavaScript, React, version control, and advanced React concepts. </td>
+      </tr>
```
Comment (jfmartinz) [inline UI-UX Design/readme.md:648]: Is this related to UI UX design? It says "Front end dev"

**Review by jfmartinz — CHANGES_REQUESTED** [review]

File: UI-UX Design/readme.md (hunk 1 above)
Comment (chaanakyaaM) [reply UI-UX Design/readme.md:648]: Only the meta's repo contains the frontend,  if you want i can remove it...

**Comment by chaanakyaaM** [comment]
done sir !

**Comment by jfmartinz** [comment]
Thanks

**Comment by chaanakyaaM** [comment]
can you add gssoc label and level1 label to this PR.
Thank you.
